stf.inv

```diff
@@ -0,0 +1,3 @@
+[stf]
```

Comment:
This file is not needed, if you include a note that people need to add an inventory file., and list the groups that are needed in that file.

You can reference a default.inv with those groups already defined, but commented hostnames, such as in [1].


[1] https://github.com/opnfv/barometer/blob/master/docker/ansible/default.inv